dao/src/main/scala/za/co/absa/enceladus/dao/MenasDAO.scala

```diff
@@ -15,62 +15,106 @@
 
 package za.co.absa.enceladus.dao
 
+import org.apache.spark.sql.types.StructType
 import za.co.absa.enceladus.model._
 import za.co.absa.atum.model._
 
+/**
+  * Trait for Menas API DAO.
```

Comment:
This was deliberate because the trait is abstract, and REST is just one implementation. I imagine in the future there might be an implementation that uses HDFS to store this information as backup, for using Menas offline or to decouple the spark jobs from Menas.